app/src/androidTest/java/com/google/jetpackcamera/ImageCaptureDeviceTest.kt

```diff
@@ -152,6 +154,147 @@ internal class ImageCaptureDeviceTest {
         Truth.assertThat(doesImageFileExist(uri, "video")).isFalse()
     }
 
+    @Test
+    fun multiple_image_capture_external() {
+        val timeStamp = System.currentTimeMillis()
+        val uriStrings = arrayListOf<String>()
+        for (i in 1..3) {
+            val uri = getTestUri(DIR_PATH, timeStamp + i.toLong(), "jpg")
+            uriStrings.add(uri.toString())
+        }
+        val result =
+            runScenarioTestForResult<MainActivity>(
+                getMultipleImageCaptureIntent(
+                    uriStrings,
+                    MediaStore.INTENT_ACTION_STILL_IMAGE_CAMERA
+                )
+            ) {
+                // Wait for the capture button to be displayed
+                composeTestRule.waitUntil(timeoutMillis = APP_START_TIMEOUT_MILLIS) {
+                    composeTestRule.onNodeWithTag(CAPTURE_BUTTON).isDisplayed()
+                }
+
+                composeTestRule.onNodeWithTag(CAPTURE_BUTTON)
+                    .assertExists()
+                    .performClick()
```

Comment:
Instead of explicitly calling performClick() and waiting for toasts 3 times, can you just do this in a `for` loop?